client/store.js

```diff
@@ -55,6 +55,16 @@ store.getCurrentAccountId = function() {
     return data.currentAccountId;
 }
 
+store.getDefaultAccountId = function() {
+    return data.settings.get('defaultAccountId');
+}
+
+store.setDefaultAccountId = function(id) {
+    //Do not use store.changeSetting so that the change is immediately visible
```

Comment:
Please do use "store.changeSetting" and make sure it dispatches a new Forward event, instead? (in particular, server errors are not handled here). Then you can remove this function.